course_catalog/constants.py

```diff
@@ -81,6 +81,15 @@ class ListType(Enum):
     LIST = "userlist"
 
 
+class StaffListType(Enum):
+    """
+    Enum for StaffList objects; sorted = staffpath, unsorted = stafflist
+    """
+
+    PATH = "staffpath"
+    LIST = "stafflist"
```

Comment:
If having separate list types for staff vs user lists is going to create problems in terms of using the same front end component for both, let me know.  I can make them the same ("path", "list") and add a database migration to convert existing userlists.

Reply:
Having `list_type = "list" | "path"` (possibly more for stafflists in the future) seems good to me. We'd probably also need to the BaseListSerializer's `object_type`, which currently reads from `list_type`. 

It is appealing...but I'm not sure it's worth the effort at the moment... If it ends up being more annoying than I expect, we could revisit?